lib/tools/adb-commands.js

```diff
@@ -1700,7 +1700,7 @@ methods.screenrecord = function screenrecord (destination, options = {}) {
  *
  * @param {string} ime - Valid IME identifier
  * @param {Function} fn - Function to execute
- * @returns {*} The result of the given function
+ * @returns {Promise<any>} The result of the given function
```

Comment:
I assume we could generify these args to point out the result would be of the same type that the passed function  returns

Reply:
yes, we could probably use generics here. I didn't look at these types too closely; trying to do the bare minimum needed to get everything working